tests/ceph_installer/test_cephadm.py

```diff
@@ -64,6 +64,9 @@ def run(ceph_cluster, **kw):
     # add all hosts
     cephadm.manage_hosts()
 
+    # Add lables to the host
+    cephadm.label_host()
```

Comment:
Can we control the flow? In other words, `add_daemons` does not use labels... it would be good if we can modify `add_daemons` to use labels or have an option to to switch between the host or labels. In this way, we can make better use of the labels option. What do you think?